enterprise/frontend/src/metabase-enterprise/collections/components/CollectionAuthorityLevelIcon.tsx

```diff
@@ -9,7 +9,7 @@ import { AUTHORITY_LEVELS } from "../constants";
 import { isRegularCollection } from "../utils";
 
 interface Props extends Omit<IconProps, "name" | "tooltip"> {
-  collection: Collection;
+  collection: Pick<Collection, "authority_level">;
```

Comment:
I chose `Pick` to reduce the code changes in other places, especially with the typing craziness going on, so `Pick` is the safe option where it still accepts objects that contain `authority_level` rather than just accepting an `authorityLevel` prop. I can change it knowing now that there's not that many changes